.github/workflows/workflow.yaml

```diff
@@ -65,7 +65,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        gatekeeper: [ "release-3.13", "release-3.14" ]
+        gatekeeper: [ "release-3.14", "release-3.15" ]
+        engine: [ "cel", "rego" ]
```

Comment:
for cel, we need to add release-3.16 here and for gator since only 3.16 has the fix for the param bug. can you pls update this such that for versions before 3.16, skip CEL engine.